core/src/main/java/io/debezium/connector/cassandra/AbstractConnectorTask.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.cassandra;
+
+import static io.debezium.connector.cassandra.CassandraConnectorConfig.CASSANDRA_NODE_ID;
+import static io.debezium.connector.cassandra.CassandraConnectorConfig.COMMIT_LOG_RELOCATION_DIR;
+import static io.debezium.connector.cassandra.CassandraConnectorConfig.SCHEMA_POLL_INTERVAL_MS;
+import static io.debezium.connector.cassandra.CassandraConnectorConfig.SNAPSHOT_POLL_INTERVAL_MS;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.connect.source.SourceRecord;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.connector.base.ChangeEventQueue;
+import io.debezium.connector.cassandra.exceptions.CassandraConnectorConfigException;
+import io.debezium.connector.cassandra.exceptions.CassandraConnectorTaskException;
+import io.debezium.connector.common.BaseSourceTask;
+import io.debezium.pipeline.ChangeEventSourceCoordinator;
+import io.debezium.pipeline.DataChangeEvent;
+import io.debezium.pipeline.spi.Offsets;
+
+abstract class AbstractConnectorTask extends BaseSourceTask<CassandraPartition, CassandraOffsetContext> {
+
+    private volatile ChangeEventQueue<DataChangeEvent> queue;
+
+    private CassandraConnectorTaskTemplate template;
+
+    @Override
+    public String version() {
+        return Module.version();
+    }
+
+    @Override
+    protected ChangeEventSourceCoordinator<CassandraPartition, CassandraOffsetContext> start(Configuration config) {
+        CassandraConnectorConfig connectorConfig = new CassandraConnectorConfig(config);
+
+        if (connectorConfig.numOfChangeEventQueues() != 1) {
+            throw new CassandraConnectorConfigException(String.format("configuration property %s must be equal to 1",
```

Comment:
@ggaborg you might explain this in this log message, explicitly mentioning that it has to be 1 under debezium server

Reply:
@smiklosovic this entire class only applies to debezium, not to standalone

Reply:
maybe I interpreted your "_ But for the second case the value of this specific property must be 1_ " incorrectly. What did you mean by "second case"?

Reply:
@smiklosovic By 'second case' I meant running the connector under Debezium Server, not as standalone. This entire AbstractConnectorTask only applies when running under Debezium Server, not as standalone.